docs/readthedocs/source/doc/Nano/QuickStart/pytorch_onnxruntime.md

```diff
@@ -86,4 +86,4 @@ y_hat = ort_model(x)
 y_hat.argmax(dim=1)
 ```
 - Note
-    `ort_model` is not trainable any more, so you can't use like trainer.fit(ort_model, dataloader)
\ No newline at end of file
+    `ort_model` is not trainable any more, so you can't use a code like `trainer.fit(ort_model, dataloader)`
```

Comment:
how about "cannot use it in `fit` such as `trainer.fit(ort_model, dataloader)`"?

Reply:
Have changed it